tests/components/calendar/test_trigger.py

```diff
@@ -82,7 +82,7 @@ async def async_get_events(
         end_date: datetime.datetime,
     ) -> list[dict[str, Any]]:
         """Get all events in a specific time frame, used by the demo calendar."""
-        _LOGGER.debug("Fetching events between %s, %s", start_date, end_date)
+        _LOGGER.debug(f"Fetching events between {start_date}, {end_date}")
```

Comment:
We don't use f-strings in logging statements (due to performance, ie to delay the operation).

Reply:
I've reverted the fstring logging, and then moved this into the main code. I found this pretty essential for debugging  so maybe it will be in production also in the future.